cli/src/factory_ops/unboxing/init

```diff
@@ -304,7 +304,16 @@ echo "Bringing HCTL cluster out of maintenance mode" 2>&1 | tee -a ${LOG_FILE}
 hctl node unmaintenance --all 2>&1 | tee -a ${LOG_FILE}
 
 echo "Ensure cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
-ensure_healthy_cluster
+
+ensure_healthy_cluster "false" "false"
```

Comment:
Please add a comment about the parameters to ensure_healthy_cluster. Ideally a function should not exit the script but just return and the invoker or the callee must decide to exit or not.

Reply:
I think let's add a jira ticket number https://jts.seagate.com/browse/EOS-15360 in the comment as this is still a workaround.